app/components/admin-dashboard.js

```diff
@@ -1,4 +1,15 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+  actions: {
+    searchDates: function() {
+      let dStart = this.get('dateStart');
+      let dEnd = this.get('dateEnd');
+      let start = Date.parse(dStart);
+      let end = Date.parse(dEnd);
+
+      console.log("dateStart:", dStart, start);
```

Comment:
lets remember to take these out - I know I asked you to push up what you had, it is just more of a reminder for myself :)